charts/internal/machine-class/templates/machine-class.yaml

```diff
@@ -55,10 +55,17 @@ providerSpec:
   dnsConfig:
 {{ toYaml $machineClass.dnsConfig | indent 4 }}
 {{- end }}
-{{- if $machineClass.memoryFeatures }}
-  memoryFeatures:
-{{ toYaml $machineClass.memoryFeatures | indent 4 }}
+{{- if $machineClass.memory }}
+  memory:
```

Comment:
You already have a field named `memory` above. I believe `cpus` and that old `memory` field are no longer needed and should be replaced by `resources`?

Reply:
good catch, and yes I can change it to resources.